frontend/Panel.js

```diff
@@ -234,7 +239,16 @@ class Panel extends React.Component {
       );
     }
     if (!this.state.isReact) {
-      return <div style={styles.loading}><h2>Looking for React…</h2></div>;
+      return (
+        <div style={styles.loading}>
+          <h2 style={styles.errorMessage}>Unable to find React…</h2>
```

Comment:
This does not necessarily mean it’s unable to find React. As you can see it could also mean it is “looking for React” (doesn’t it always take a while to connect?) Especially with React Native.

We need to avoid showing a strongly negative message when we still might get a connection. Can you distinguish loading state from failed state somehow? Or maybe reword to be more ambiguous and imply it takes a few seconds to load if React is there.